### PR TITLE
Create library function for Hadoop properties

### PR DESCRIPTION
**Pull request by wolf31o2**
Move processing of Hadoop properties into a reusable library.

### CODE REVIEW

File: libraries/helpers.rb [hunk 1]
```diff
@@ -116,6 +116,19 @@ def hadoop_lib_dir
     end
 
     #
+    # Support deprecated Hadoop properties
+    #
```
Comment (dereklwood) [inline libraries/helpers.rb]: usage comment would help... took me a while to figure out what this was doing

**Comment by dereklwood** [comment]
non-blocking minor comments, 👍 lgtm

File: libraries/helpers.rb (hunk 1 above)
Comment (wolf31o2) [inline libraries/helpers.rb]: Good idea.

**Comment by dereklwood** [comment]
👍 lgtm

**Pull request merged**